lap3dkernel/c++SIMD/VCL_AgnerFog_version1/vectorf512.h

```diff
@@ -1379,6 +1379,11 @@ static inline Vec8d sqrt(Vec8d const & a) {
     return _mm512_sqrt_pd(a);
 }
 
+// aprrox 1/sqrt(a) function, takes double type as function argument

```

Comment:
VCL version 1 or version 2 don't have simd approx rsqrt for double type argument, so I added to the local VCL lib.